skiko/src/commonTest/cpp/TestHelpers.cc

```diff
@@ -0,0 +1,19 @@
+#include "common.h"
+
+SKIKO_EXPORT void org_jetbrains_skiko_tests_TestHelpers__1nFillByteArrayOf5(KNativePointer byteArray) {
+    char *bytes = reinterpret_cast<char*>(byteArray);
+    bytes[0] = 1;
+    bytes[1] = 2;
+    bytes[2] = 3;
+    bytes[3] = 4;
+    bytes[4] = 5;
+}
+
+SKIKO_EXPORT void org_jetbrains_skiko_tests_TestHelpers__1nFillFloatArrayOf5(KNativePointer floatArray) {
```

Comment:
Shalln't we test all types here (ShortArray, CharArray)?

Reply:
Sure. Added tests for ints, doubles, shorts.

CharArray is not clear yet, since there is no toInterop(charArray). Although there is `toInterop(string)` but it's not the same on all platforms.
So maybe we need to have all pairs of methods (toInterop, fromInterop) for every type? Although not many of existing methods are actually used in the skiko code.